frontend/next.config.ts

```diff
@@ -57,6 +57,19 @@ const nextConfig: ExplorerConfig & NextConfig = {
   experimental: {
     externalDir: true,
   },
+  images: {
+    domains: [
+      "cloudflare-ipfs.com",
+      "ipfs.fleek.co",
+      "nearnaut.mypinata.cloud",
+      "arweave.net",
+      "beerpunks.mypinata.cloud",
+      "bafybeidw4zstz4q3gezdbtslzvxkhkjko32octdgg2xbtsghjhsj7xkd7q.ipfs.dweb.link",
+      "nftstorage.link",
+      "ipfs.io",
+      "ewtd.mypinata.cloud",
+    ],
```

Comment:
This list of domains could be Infinite, so I think we need to drop  next/image `<Image />` component

Reply:
@shelegdmitriy Most of these domains are just gateways to IPFS, so it is expected that they take some time to load (we need to keep retrying on error and communicate that to the users - this might be done in the future PRs)